.env.example

```
@@ -86,3 +86,17 @@ HE4RT_PRESENTATION_ROLE_ID=
 HE4RT_DELAS_CHANNEL_ID=
 HE4RT_COMITE_DELAS_ROLE_ID=
 HE4RT_DELAS_ROLE_ID=
+
+BACKUP_ENABLED=false
+BACKUP_DISKS=local
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The BACKUP_DISKS key should go before the BACKUP_ENABLED key

(UnorderedKey)